Helper/InsightsHelper.php

```diff
@@ -165,7 +168,7 @@ public function setAuthenticatedUserToken(Customer $customer): string|null
     public function setNonAuthenticatedUserToken(): void
     {
         $metaData = $this->cookieMetadataFactory->createPublicCookieMetadata()
-            ->setDuration($this->configHelper->getCookieLifetime())
+            ->setDuration(InsightsHelper::ALGOLIA_USER_TOKEN_DEFAULT_LIFETIME)
```

Comment:
@damcou Won't this perpetually renew? If so not sure if that is the desired behavior? My feeling is we should just leave the anonymous cookie alone. Perhaps however we should explicitly delete it on logout which will result in the Insights lib generating a fresh `_ALGOLIA` cookie. That probably makes the most sense. WDYT?

Reply:
@cammonro You're right, it's renewed at each page load ... Since it's not clear what should be done with the `_ALGOLIA` token, I'll remove this behaviour and remove it on logout as you suggest ;)